src/main/java/br/com/zup/concessionariacitroen/services/EstoqueService.java

```diff
@@ -0,0 +1,20 @@
+package br.com.zup.concessionariacitroen.services;
+
+import br.com.zup.concessionariacitroen.models.Automovel;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class EstoqueService {
+
+    private static List<Automovel> ListaAutomoveis = new ArrayList<>();
+    private int quantidade;
+
+    public List<Automovel> visulizarAutomoveis(){
```

Comment:
Alterar o tipo do carro;
```suggestion
    public List<Carro>
```
O método deve ser de visualizar o estoque, então, renomear o método.
```suggestion
    public List<Automovel> visulizarEstoque(){
```
É necessário sobrescrever o toString da model Estoque, para imprimir corretamente.